pages/linux/sic.md

```diff
@@ -0,0 +1,29 @@
+# sic
+
+> simple irc client.
+> Part of suckless.org tools.
+> More information: <https://tools.suckless.org/sic/>.
+
+- Connect to host (default: irc.oftc.net) with nickname (default: $USER):
+
+`sic -n {{nickname}}`
+
+- Connect to a different host with PASS connection password:
+
+`sic -h {{host}} -n {{nickname}} -k {{password}}`
+
+- Usage with highlighted alerts on mention of your username:
+
+`sic | awk '/username/ {printf "\a"}1'`
+
+- Join a channel (stdin):
```

Comment:
Not sure what you mean by `(stdin)` here, or in future examples.

Reply:
`sic` works on `stdin` input. Please see [usage](https://tools.suckless.org/sic/usage/) and/or the man page of `sic`.

Reply:
@Waples I see what you mean—this is a TUI and therefore when you start the tool it offers a prompt for you to enter commands etc.. So just omit `(stdin)` here and in the next examples.

See other documented TUIs for reference, e.g.:
- [`rainbowstream.md`](https://github.com/tldr-pages/tldr/blob/master/pages/common/rainbowstream.md)
- [`rtv.md`](https://github.com/tldr-pages/tldr/blob/master/pages/common/rtv.md)
- [`vim.md`](https://github.com/tldr-pages/tldr/blob/master/pages/common/vim.md)

Reply:
Thanks @bl-ue, i'll implement that.